### DISCUSSION TITLE
Improving Spanish names + Adding Catalan names

### DISCUSSION THREAD

**Discussion by xmontero** [benkeen/generatedata#770] [Ideas]
Hi, I discovered this awesome project today.

I'm from Barcelona, Spain and I found that the Spanish names are not so realistic.

I want to contribute to the project improving the Spanish name generator.

Additionally we have another language here: Catalan, which is the "main language" for people in Barcelona and catalan names for people are different.

Before spending time... How welcome are those contributions? If I contribute and do a PR, should I expect it to be merged?

**Comment by benkeen** [comment]
Hey @xmontero, thanks for reaching out.  

Those contributions would be _very_ welcome! For translations I'll be able to merge them quickly, but it may take a week or so to get onto the main website. 

Ben

**Comment by xmontero** [comment]
I forked it and I see that building/running it seems rather more complicated than I expected. I didn't expect it to require databases and so... I thought it was more a frontend thing or with a simple-basic backend (no databases involved).

I'm not very sure about secondary effects of launching it...

So I'd probably edit the `.json` files directly without running it locally.

Couple of questions:

1. About adding the catalan language here https://github.com/benkeen/generatedata/discussions/772
2. About some existing spanish translations here https://github.com/benkeen/generatedata/discussions/773

**Comment by benkeen** [comment]
No problem, I'll check out the translation once it's complete.
